nightly/pytest-sanity.txt

```diff
@@ -150,7 +150,7 @@ pytest --timeout=600 sanity/split_storage.py --features nightly
 # Test for resharding
 pytest --timeout=120 sanity/resharding.py
 pytest --timeout=120 sanity/resharding.py --features nightly
-
-# Test for resharding error handling
 pytest --timeout=120 sanity/resharding_error_handling.py
 pytest --timeout=120 sanity/resharding_error_handling.py --features nightly
+pytest --timeout=120 sanity/rpc_tx_resharding.py
+pytest --timeout=120 sanity/rpc_tx_resharding.py --features nightly
```

Comment:
nit: I would rename to resharding_* so that the files are close to each other in IDE

Reply:
Changed name